DEPS

```diff
@@ -23,6 +23,7 @@ deps = {
   "third_party/bitcoin-core/src": "https://github.com/bitcoin/bitcoin.git@95ea54ba089610019a74c1176a2c7c0dba144b1c",
   "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
   "third_party/rapidjson/src": "https://github.com/Tencent/rapidjson.git@06d58b9e848c650114556a23294d0b6440078c61",
+  "third_party/rust/cxx/v1/crate": "https://github.com/dtolnay/cxx.git@79b90b410573380bbdd84723cc220323456c2290",
```

Comment:
we can't do this, we need upstream's patches

Reply:
Reverted.

Reply:
can't we just copy these patches and apply ourselves via apply_patches infrastructure?

Reply:
Like I mentioned somewhere else, the cxx sources are also pushed on Chromium, from where I lifted it from. A process to have a DEPS checked out, and patched, although not impossible, would require certain planning, specially as the dependency would seem to have a dirty tree.

I would advise to have this looked into outside the version upgrade PR, as I suspect that we will block the PR further with this rust issue.